src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -407,30 +497,20 @@ def launch_fluent(
             )
         )
     if start_instance:
```

Comment:
@prmukherj maybe you're already doing it. lines 491 to 499 can collapse to `if _start_instance(start_instance):`

Reply:
@seanpearsonuk, this has been applied already. Thanks for pointing this out. Will push it shortly